analysis/parameter_tuning.py

```diff
@@ -153,6 +179,35 @@ def _find_candidates(histogram: histograms.Histogram) -> List:
         max_contributions_per_partition=linf_bounds)
```

Comment:
btw, which value does it take if we provide None? I mean for tuning we need to specify some value, we can't not specify l0 or linf. Is it specified by a user in this case?